core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -668,42 +669,37 @@ private boolean handleKeycode(int keycode, boolean notificationButton) {
                 }
                 return false;
             case KeyEvent.KEYCODE_MEDIA_NEXT:
-                if (getStatus() != PlayerStatus.PLAYING && getStatus() != PlayerStatus.PAUSED) {
-                    return false;
-                } else if (notificationButton || UserPreferences.shouldHardwareButtonSkip()) {
-                    // assume the skip command comes from a notification or the lockscreen
-                    // a >| skip button should actually skip
+                if (!notificationButton) { // hardware button
+                    return handleKeycode(UserPreferences.getHardwareForwardButton().keyCode, true);
```

Comment:
Could you please add a comment like the one below here? I was a bit confused when reading this.

```
// Handle remapped button as notification button which is not remapped again.
```